Assets/MixedRealityToolkit/Definitions/SceneSystem/LightingSceneTransitionType.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.SceneSystem
+{
+    /// <summary>
+    /// Used by scene service to control how to transition from one lighting scene to another.
+    /// </summary>
+    public enum LightingSceneTransitionType
+    {
+        None,           // Previous lighting scene is unloaded, new lighting scene is loaded. No transition.
```

Comment:
What is a lighting scene?

Reply:
The Lighting scene is a largely empty scene which controls lighting settings.
Ambient lighting, skybox, sun direction, etc. A default lighting scene is loaded on initialization.
After that the active lighting scene may be changed at any time via SetLightingScene.
Only one lighting scene can ever be loaded at a time.